src/components/DiagramFrame/SeqDiagram/LifeLineLayer/Participant.vue

```diff
@@ -52,9 +52,10 @@ export default {
     const translate = computed(() => {
       let top = intersectionTop.value + scrollTop.value
       if (intersectionTop.value > INTERSECTION_OFFSET && store?.state.stickyOffset) top += store?.state.stickyOffset
-      if (top < props.offsetTop + PARTICIPANT_TOP) return 0
-      const diagramHeight = store?.state.diagramElement.clientHeight || 0
-      return Math.min(top - PARTICIPANT_TOP, diagramHeight - PARTICIPANT_HEIGHT) - props.offsetTop
+      const diagramHeight = store?.state.diagramElement?.clientHeight || 0
+      const diagramTop = store?.state.diagramElement ? getElementDistanceToTop(store?.state.diagramElement) : 0
+      if (top < props.offsetTop + diagramTop) return 0
+      return Math.min(top - diagramTop, diagramHeight - PARTICIPANT_HEIGHT) - props.offsetTop
```

Comment:
Do we not need to take participant top into consideration at all?
![image](https://github.com/mermaid-js/zenuml-core/assets/169417/b00669d6-407b-4eb1-a8a7-ea7c582bdec3)

Reply:
It already be considered as the `props.offsetTop`